app/assets/javascripts/iht.js

```diff
@@ -115,6 +115,19 @@ if($('[data-enhanced-select]').length > 0){
   numberInputs();
 
 
+
+// =====================================================
+// Handle the IHT UR panel dismiss link functionality
+// =====================================================
+    var cookieData=GOVUK.getCookie("iht_ur_panel");
+    if (cookieData == null) {
+        $("#ur-panel").css("display", "block");
```

Comment:
Best practice is to assign a class (such as url-panel--visible) rather than setting styles with js